Cargo.toml

```diff
@@ -11,6 +11,7 @@ repository = "https://github.com/sorear/smetamath-rs"
 clap = "2.5.2"
 filetime = "0.1.10"
 fnv = "1.0.2"
+regex = "0.1.71"
```

Comment:
I can't prove this is the guilty line, but this PR raises the binary size by 52%, from 1179440 bytes to 1802032 bytes.  Requires discussion if that's acceptable, although `clap` is also huuuge :/